### ISSUE TITLE
Is this repository still maintained?

### ISSUE THREAD

**Issue by burgerga** [serhii-londar/open-source-mac-os-apps#927]
If not perhaps we can help by moving it to an org so multiple people can maintain, or autogenerating the README.md?

**Comment by sundeepdwd** [comment]
**"macos" "linux" "releases" site:github.com -gist dmg** in google search is also good option 
[link](https://www.google.com/search?sca_esv=a03cc0aa586dcb1e&rlz=1C5CHFA_enNP1094NP1094&biw=1680&bih=928&sxsrf=AHTn8zqcjj7M35ZWh8LfQmCR0IIIaJEJOQ:1738224543339&q=%22macos%22+%22linux%22+%22releases%22+site:github.com+-gist+dmg&source=lnms&fbs=ABzOT_CWdhQLP1FcmU5B0fn3xuWpA-dk4wpBWOGsoR7DG5zJBjLjqIC1CYKD9D-DQAQS3Z5NmlRTZM9mMZwadeXOSzf-GqFV4lDd4wAv8PNxid8J4e_yfHMsIOie1ONPtKN8yJ84bRUJtR_9M4vxUTrwAep3GpUrFFtZXXBJgstu90om1O8VJNo&sa=X&ved=2ahUKEwji-MeD_5yLAxUO1TgGHVBJBLgQ0pQJegQIExAB)

**Comment by serhii-londar** [comment]
@burgerga Yes, I'm checking it from time to time when I have time. Currently I plan to do clean up and probably improve readme structure. Please let me know If you have some ideas.
